### PR TITLE
opencv-haar-classifier training using opencv 3.2.0

### PR DESCRIPTION
**Pull request by jayluxferro**
opencv 3.2.0

### CODE REVIEW

**Comment by mrnugget** [comment]
First of all: thanks for taking the time and the effort to do this! I'm super happy to get PRs that improve this repo. But I have a few issues with this PR:

1. The existing solution is for OS X/macOS only. This PR turns it into Linux only. Why? Does it not work on macOS?
2. The acknowledgements and references are removed. Why?
3.  I don't think a separate install script it's necessary. I don't see the it as a responsibility of this tutorial to make sure that OpenCV is properly installed. Mentioning something like "be sure to use compile flags X, Y, Z" would be fine though.
4. The URL of the git repo is changed in the README. Why?

In short: I'd love to see the **most minimal changes possible to this tutorial so it works with OpenCV 3.2.0** if we're saying this tutorial supports it.
